napalm_iosxr/__init__.py

```diff
@@ -13,4 +14,14 @@
 # the License.
 
 """napalm_iosxr package."""
-from iosxr import IOSXRDriver
+
+# Import stdlib
+import pkg_resources
+
+# Import local modules
+from iosxr import IOSXRDriver  # noqa
```

Comment:
Should change to:

```
from napalm_iosxr.iosxr import IOSXRDriver
```

The original import is not valid in Python3. I also would drop the # noqa comment and do.

```
__all__ = ['IOSXRDriver']
```